tests/databases/test_bigquery.py

```diff
@@ -203,6 +204,54 @@ def test_load_file_to_table(database_table_fixture):
     test_utils.assert_dataframes_are_equal(df, expected)
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "database_table_fixture",
+    [
+        {
+            "database": Database.BIGQUERY,
+            "table": Table(metadata=Metadata(schema=SCHEMA)),
+        },
+    ],
+    indirect=True,
+    ids=["bigquery"],
+)
+def test_load_file_to_table_natively_for_not_optimised_path(database_table_fixture):
+    """Test loading on files to bigquery natively for non optimized path."""
```

Comment:
@sunank200  Why is this test needed?

Reply:
This is unit test for `load_file_to_table_natively`